src/assets/scss/material.palettes.scss

```diff
@@ -122,6 +122,7 @@ $seamlessaccess-blue: #216e93;
 // state-notice zepling colors
 $state-notice-dark: #ff9c00;
 
+$state-notice-darkest: #ff6400;
```

Comment:
also need to add this to the foreground palette here 

https://github.com/ORCID/orcid-angular/blob/296ec752215894a6653e41d392f88ce3538ed897/src/assets/scss/material.light-theme.scss#L43

Reply:
Done!